.github/workflows/test.yml

```diff
@@ -370,7 +370,7 @@ jobs:
           AIRFLOW__COSMOS__REMOTE_TARGET_PATH: "s3://cosmos-remote-cache/target_compiled/"
           AIRFLOW__COSMOS__REMOTE_TARGET_PATH_CONN_ID: aws_s3_conn
           DBT_PROJECT_NAME: "altered_jaffle_shop"  # The syntax of jaffle_shop is not supported in dbt 1.5.4, only in dbt >= 1.10
-          TEST_SINGLE_DAG: "basic_cosmos_task_group.py"
+          TEST_SINGLE_DAG: "basic_cosmos_task_group.py"  # There are circumstances when we only need to run a single DAG, but we were parsing all the DAGs. This change is to avoid parsing all the DAGs.
```

Comment:
It would be nice to check if we can drop support for dbt 1.5

Currently, we're testing only one DAG for dbt 1.5 for a known case we want to test, but I guess it makes it hard to remember the exact reasons/changes we wanted to test across versions.